.github/workflows/release.yml

```diff
@@ -29,14 +29,18 @@ jobs:
         distribution: 'temurin'
         java-version: ${{ matrix.java }}
         cache: 'maven'
+    - name: Log in to GitHub Container Registry
```

Comment:
These configurations align with my current understanding of the release workflow. We will also need to ensure we have these credentials backed up in a "secondary" vault, which we can chat about offline.

Reply:
OK.  I tried to follow your pattern here and I think you had just followed the recommendation of the GitHub docs.  The alternative to using the injected GITHUB_TOKEN is to use a Personal Access Token (PAT).
I'm not sure that either needs to be stored outside of GitHub but I'm willing to hear the argument.

P.S. a related topic for us to chat about is whether we should set "restrictive" permissions for our GITHUB_TOKEN at the org level.  https://docs.github.com/en/actions/security-guides/automatic-token-authentication